src/validation/transaction.cairo

```diff
@@ -1,22 +1,20 @@
 //! Transaction validation helpers.
 
-use crate::types::transaction::{Transaction, TransactionTrait};
+use crate::types::transaction::{Transaction};
+use crate::utils::hash::Hash;
 
 /// Validate transaction and return transaction fee.
 ///
 /// This does not include script checks and outpoint inclusion verification.
 pub fn validate_transaction(
```

Comment:
I don't think it's worth introducing an argument just to use it in error message.
For debugging purposes we can just println txid before calling this function.